README.md

```diff
@@ -17,68 +18,68 @@ On the frontend, use the [Co-Authors Plus template tags](http://vip.wordpress.co
 
 This plugin is an almost complete rewrite of the Co-Authors plugin originally developed at [Shepherd Interactive](http://www.shepherd-interactive.com/) (2007). The original plugin was inspired by the 'Multiple Authors' plugin by Mark Jaquith (2005).
 
-== Frequently Asked Questions ==
+## Frequently Asked Questions 
 
-= How do I add Co-Authors Plus support to my theme? =
+* How do I add Co-Authors Plus support to my theme?
 
-If you've just installed Co-Authors Plus, you might notice that the bylines are being added in the backend but aren't appearing on the frontend. You'll need to [add the template tags to your theme](http://vip.wordpress.com/documentation/incorporate-co-authors-plus-template-tags-into-your-theme/) before the bylines will appear.
+> If you've just installed Co-Authors Plus, you might notice that the bylines are being added in the backend but aren't appearing on the frontend. You'll need to [add the template tags to your theme](http://vip.wordpress.com/documentation/incorporate-co-authors-plus-template-tags-into-your-theme/) before the bylines will appear.
```

Comment:
I'm not totally sold on the `>`/quote indenting here. I think a regular paragraph would be fine